### PR TITLE
Removes Syndicate Infiltrator Gamemode

### PR DESCRIPTION
**Pull request by VaelophisNyx**
# Document the changes in your pull request

Reverts:
https://github.com/yogstation13/Yogstation/pull/14846
https://github.com/yogstation13/Yogstation/pull/14795 (Only the infil stuff)
https://github.com/yogstation13/Yogstation/pull/14388
https://github.com/yogstation13/Yogstation/pull/12850
https://github.com/yogstation13/Yogstation/pull/12270

Should have stayed in the TG trash can, where it belongs.

Bad gamemode, mass majority would support this removal.

# Wiki Documentation

Mentions of Infils need to be removed from the wiki antagonists page

# Changelog

:cl:  
rscdel: Syndicate Infiltation Gamemode & All Related Content Is Removed.
/:cl:

### CODE REVIEW

**Comment by ilikesaying** [comment]
9A3C308C-18F4-463D-A07C-2072793FCB7B

**Comment by Cartlord** [comment]
I have genuinely never known that it was an Infiltrator round. They MIGHT be good for Dynamic because then they have to contend with other shit happening, but they're a horrible standalone.

**Comment by VaelophisNyx** [comment]
you knwo what's worse, Cart?

fucking
infils
had
no
dynamic
ruleset

nor any of the framework to make that work

this shit was a dumpster fire from the start

**Comment by VaelophisNyx** [comment]
ok sorry now they DO have one but no, infiltrators is a TERRIBLE game mode even for a midround
it shouldn't exist

File: code/modules/mob/living/silicon/ai/decentralized/management/ai_controlpanel.dm [hunk 1]
```diff
@@ -197,13 +197,6 @@ GLOBAL_VAR_INIT(ai_control_code, random_nukecode(6))
 		data["current_ai_ref"] = REF(user)
 
 	data["can_log_out"] = !one_time_password_used
-
-	for(var/mob/living/silicon/ai/A in GLOB.ai_list)
```
Comment (Bibby0110) [inline code/modules/mob/living/silicon/ai/decentralized/management/ai_controlpanel.dm:201]: you uhh, can't just remove this, just remove the hijacking part

File: tgui/packages/tgui/interfaces/AiControlPanel.js [hunk 2]
```diff
@@ -125,7 +125,7 @@ export const AiControlPanel = (props, context) => {
                           buttons={(
                             <Fragment>
                               <Button color={ai.can_download ? "good" : "bad"} tooltip={!data.intellicard ? ai.can_download ? "Requires IntelliCard" : "&¤!65%" : null} disabled={data.intellicard ? !ai.can_download : true} icon="download" onClick={() => act("start_download", { download_target: ai.ref })}>{ai.can_download ? "Download" : "&gr4&!/"}</Button>
-                              {!!data.is_infiltrator && !ai.being_hijacked && (
+                              {!ai.being_hijacked && (
                                 <Button color="good" tooltip="Requires serial exploitation unit" icon="download" onClick={() => act("start_hijack", { target_ai: ai.ref })}>Start hijacking</Button>
```
Comment (Bibby0110) [inline tgui/packages/tgui/interfaces/AiControlPanel.js]: these lines are left over

File: code/modules/mob/living/silicon/ai/decentralized/management/ai_controlpanel.dm (hunk 1 above)
Comment (VaelophisNyx) [reply code/modules/mob/living/silicon/ai/decentralized/management/ai_controlpanel.dm:201]: whoops

Comment (VaelophisNyx) [reply code/modules/mob/living/silicon/ai/decentralized/management/ai_controlpanel.dm:201]: will be fixed next push

**Review by JamieD1 — CHANGES_REQUESTED** [review]
I will support disabling it, but dont yeet it.

**Pull request closed without merging**